composer.json

```
@@ -0,0 +1,46 @@
+{
+    "name": "alymosul/laravel-exponent-push-notifications",
+    "description": "Exponent push notifications driver for laravel",
+    "homepage": "https://github.com/alymosul/laravel-exponent-push-notifications",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Aly Suleiman",
+            "email": "alymosul@gmail.com",
+            "role": "Developer"
+        }
+    ],
+    "require": {
+        "php": "^7.1.3",
+        "alymosul/exponent-server-sdk-php": "1.1.*",
+        "laravel/framework": "^5.6 || ^6.0"
+    },
+    "require-dev": {
+        "mockery/mockery": "^0.9.5",
+        "phpunit/phpunit": "~5.4||~5.7||~6.0||^7.0",
+        "orchestra/testbench": "~3.3||~3.4||~3.5||~3.6||~3.7"
```

Support for v4 needed for laravel 6 s well.
```suggestion
        "orchestra/testbench": "~3.3||~4.0"
```